contrib/cron-jobs/update-geoip-data

```diff
@@ -1 +0,0 @@
-../../intelmq/bots/experts/maxmind_geoip/update-geoip-data
```

Comment:
please leave it as is for backwards-compatibility

Reply:
Done.